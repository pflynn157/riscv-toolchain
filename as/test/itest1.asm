addi x1, x2, 20
addi x1, x4, 10
addi x2, x20, 30
