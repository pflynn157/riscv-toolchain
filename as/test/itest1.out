01410093
00a20093
01ea0113
